deployment/systemd/setup.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Radioan Service Setup
```

Comment:
"Setup" makes me think of something that only runs once and not a system service. Perhaps Config is more appropriate?

Reply:
I kept going back and forth between them. Maybe there's a better name than both of them! I'll have a think.